test/sqlsmith/mzcompose.py

```diff
@@ -150,6 +150,7 @@
     "out of valid range",
     '" does not exist',  # role does not exist
     "csv_extract number of columns too large",
+    "coalesce types text and oid cannot be matched",  # with ACL-related functions
```

Comment:
We apparently need the reverse one as well: `coalesce types oid and text cannot be matched` (like lines 140-141, 145-146)

https://buildkite.com/materialize/nightlies/builds/2933

Reply:
ok will do